src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  padding-right: 5px;
+  width: 96px;
+  justify-content: space-between;
```

Comment:
Don't fix container size (if there is no such a requirement). The width of the `.stars` container is fixed, which may not be needed. Ensure that the content can dictate its size to avoid overflow or accidental scroll bars.